ui/src/app/cohort-review/participant-status/participant-status.component.css

```diff
@@ -0,0 +1,7 @@
+.participant-status-margin-top {
```

Comment:
Lets not put the properties in the class names - if we want to add / remove more properties from the class (which is likely) then that becomes either cumbersome or odd.  Instead name the classes for what they are - so this would be `participant-id` (or even `sidebar-header`) and `demographics-info` (or something like that).

Reply:
Done